src/style.css

```diff
@@ -0,0 +1,108 @@
+html {
+  --main-text-color: #060b35;
+  --secondary-text-color: #616070;
+  --button-color: #00acdc;
+  font-family: "Roboto", sans-serif;
+  color: var(--main-text-color);
+  font-size: 10px;
+  line-height: 14px;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: white;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  margin: 0 3px 40px;
+}
+
+.card__name {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  margin: 0 0 4px;
+}
+
+.card__code {
+  color: var(--secondary-text-color);
+  margin: 0 0 16px;
+  font-weight: 400;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 0 24px;
+}
+
+.stars {
```

Comment:
Create separate files for BEM blocks
![image](https://user-images.githubusercontent.com/96409563/232689459-dcf4e095-94ee-47ce-ae44-5811f6477d70.png)